plugins/reporters/fossid/src/main/resources/templates/asciidoc/fossid_snippet.ftl

```diff
@@ -29,6 +29,7 @@
 List of all the provenances with their files and snippets.
 [#list ortResult.scanner.scanResults as scanResult]
 
+[#if scanResult.scanner.name != "FossId"] [#continue][/#if]
```

Comment:
Nit: For one-liners like this, I believe we should either also add a space after `[#continue]`, or omit the one before. In any case, be consistent WRT spacing. Or make it a three-liner with indentation 😉